src/identities.rs

```diff
@@ -79,6 +81,22 @@ where
     }
 }
 
+///Implements Zero as the empty string
+#[cfg(feature = "std")]
+impl<'a> Zero for Cow<'a, str> {
+
+    fn is_zero(&self) -> bool {
+        match self {
+            Self::Borrowed(str) => str.len() == 0,
+            Self::Owned(string) => string.len() == 0
+        }
+    }
+
+    fn zero() -> Self {
+        Self::Borrowed("")
```

Comment:
FWIW, the [default](https://doc.rust-lang.org/std/borrow/enum.Cow.html#impl-Default) would create an owned (empty) `String`.

Reply:
Fair. I guess my brain had the idea that using `Borrowed` would be the most performant, but on reevaluation, that's obviously not true.